chanreg.md

```diff
@@ -1,228 +1,191 @@
-[community registration](#community-registration).
-
-### Informal channels
-
-Informal channels that are not already taken can be registered by any user in
-it who has channel operator status, which is usually the first user to join an
-empty, unregistered channel. In this case you can use the `REGISTER` command
-in chanserv to register it to your account. See `/msg ChanServ help REGISTER`
-for details. If a channel is unregistered but nobody has operator status,
-please contact network staff and we shall see if we can find a solution that
-is acceptable for the local community.
-
-## Project registration
-
-Project registration allows your project or organisation to have an official
-representation on our services and relationship with us.
-
-### On-topic projects
+When joining an unregistered and empty channel, you will automatically be
+given channel operator access on the channel. After verifying that your
+channel is in the correct namespace, you should
+[register it with ChanServ](/guides/creatingchannels) in order to establish
+long-term control over the channel. We recommend that you choose the
+[appropriate namespace](#channel-namespaces) to avoid a disruptive move later.
+
+Should the channel already exist but is
+[expired or otherwise unused](#expiration-and-disputes), Libera.Chat staff may
+be able to assist you in taking over the channel. For channels in the on-topic
+project and community namespace, this may involve proceeding with a
+[formal group registration](#formal-group-registration) with us. For informal
+channels, we will check if it is expired and if there is consensus within the
+channel's existing community for you to take it over.
+
+Once your channel is registered with ChanServ, regardless of channel
+namespace, you are free to use it in accordance with
+[network policy](/policy). You do **not** need the explicit approval of
+Libera.Chat staff in order to register a channel and start using it on our
+network. Formal group registration is **not** required to make use of channels
+in the on-topic project and community channel namespace, and we recommend that
+new initiatives do not immediately attempt a formal group registration.
+
+## Channel namespaces
+
+Our [policies](/policy) outline that channels on Libera.Chat fall into one of
+three categories and two namespaces. Channels that begin with only a single
+`#` character are for on-topic project and community channels (e.g. `#linux`).
+Channels that begin with two or more `#` characters are for informal channels
+(e.g. `##chat`).
+
+### What is on-topic?
 
 Projects considered to be on-topic for Libera.Chat are primarily free and
 open-source software projects, and other peer-directed projects, for instance
 Linux User Groups (LUGs), student societies, hacker- or makerspaces and other
-collaborative efforts. It can also cover projects/companies of general
+collaborative efforts. It can also cover projects or companies of general
```

Comment:
While we use "can" here, do we want to add emphasis about the discretionary nature? ie
"It can, at our discretion, also cover projects or companies of general interest to our users."